tests/functionaltests/UXTestHelpers.h

```diff
@@ -81,4 +84,22 @@ class XamlEventSubscription {
     StrongId<WXDependencyProperty> _propertyToObserve;
 }; // class XamlEventSubscription
 
+// Class used to signal and wait during a test run
+class UXEvent {
```

Comment:
I can add the constructor UXEvent(autoReset) but using the "factory-like" methods to create UXEvents runs into issues because I'm removed the CC/operator=. As we discussed, I'll keep the constructor public with the enum for Manual vs Auto.

---
In reply to: [98241423](https://github.com/Microsoft/WinObjC/pull/1855#discussion_r98241423) [](ancestors = 98241423)